jaeger-core/src/main/java/io/jaegertracing/internal/JaegerTracer.java

```diff
@@ -72,10 +73,12 @@
   private final Map<String, ?> tags;
   private final boolean zipkinSharedRpcSpan;
   private final ScopeManager scopeManager;
+  private final BaggageRestrictionManager baggageRestrictionManager;
   private final BaggageSetter baggageSetter;
   private final boolean expandExceptionLogs;
+  private final TracingFactory tracingFactory;
 
-  private JaegerTracer(
+  protected JaegerTracer(
```

Comment:
I don't see why you need to duplicate constructors - it was private previously, nobody called it externally, so it should be ok to change its API

Reply:
Good point. I'll remove this one and any others I find that were originally `private` or package scope.